content/the-road-to-ember-4-0.md

```diff
@@ -0,0 +1,89 @@
+- `willTransition` and `didTransition` in favor of router service events. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_deprecate-router-events).
+- Computed Property `volatile()` calls are removed in favor of native getters. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_computed-property-volatile).
+- `this.$()` and other jQuery APIs are deprecated in favor of native brower equivalents. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_jquery-apis). An optional feature which restored this and other jQuery-specific features is also removed. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_optional-feature-jquery-integration).
+- `{{partial}}` is removed in favor of template-only components. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_ember-partial).
+- Using the built-in global resolver (`App.FooController` anyone?) is deprecated in favor of using [ember-resolver](https://github.com/ember-cli/ember-resolver), already the default for Ember CLI generated apps. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_ember-deprecate-globals-resolver).
+- Ambiguous references to a component's properties are removed, aka you must know write `{{this.someProp}}`. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_this-property-fallback).
+- `renderTemplate` is removed in favor of `{{in-element}}` or other rendering target redirection like [ember-wormhole](https://github.com/yapplabs/ember-wormhole). [Guide here](https://deprecations.emberjs.com/v3.x/#toc_route-render-template).
+- Support for the `Ember` global on `window` is removed in favor of importing the `Ember` object or using the module-based API. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_ember-global).
+- Support for features of the `<LinkTo`, `<Input`, and `<TextArea` components which are coupled to `Ember.Component` are removed. See guides on [positional arguments](https://deprecations.emberjs.com/v3.x/#toc_ember-glimmer-link-to-positional-arguments), [legacy arguments](https://deprecations.emberjs.com/v3.x/#toc_ember-built-in-components-legacy-arguments), [legacy HTML attributes](https://deprecations.emberjs.com/v3.x/#toc_ember-built-in-components-legacy-attribute-arguments), and [importing legacy built-in components](https://deprecations.emberjs.com/v3.x/#toc_ember-built-in-components-import).
+- Finally, **Ember classic is deprecated in favor of Ember Octane**. Although
+many APIs from Ember classic (like `EmberComponent`) continue to be available,
+the optional features and application configuration which define Ember Octane
+must be enabled in 4.0. See [the deprecation guide
+entry](https://deprecations.emberjs.com/v3.x#toc_editions-classic) and
+[upgrading to Ember
+Octane](https://guides.emberjs.com/release/upgrading/current-edition/) guide for
+more details.
+
+Scanning this list, the trend is that many rarely used APIs dating back to Ember 1.x patterns are removed.
```

Comment:
An optional suggestion. I think this sentence could carry a softer, less certain tone (with regards to the claimed trend that the APIs are rarely used).

```suggestion
We can see an interesting trend in the list above. The APIs that will be removed tend to be those that date back to Ember 1.x and are rarely used now (or should be rarely used).
```

Reply:
Again, made an edit